src/form-generator/form-handler-api-web-forms.ts

```diff
@@ -0,0 +1,118 @@
+    this.formRequiredFieldsParagraph = new FormRequiredFieldsParagraph();
+  }
+
+  createFormElement = async (): Promise<HTMLDivElement> => {
+    const container = new DivCreatorWithClassName().createDiv(
+      classNames.container
+    );
+    const title = new DivCreatorWithClassName().createDiv(
+      classNames.titleOneStep
+    );
+    title.textContent = config.titleOneStep.textContent;
+
+    const form = new HtmlTagName().formCreator();
+    form.id = "contact-form";
+
+    const elements = [...config.selects, ...config.fields];
+
+    for (const element of elements) {
+      if (element.name !== "Password" && element.name !== "Confirm Password") {
+        await this.formElementCreation.create(form, element);
+      }
+    }
+
+    const button = this.createSubmitButton(container);
+    form.appendChild(button);
+
+    container.appendChild(title);
+    container.appendChild(form);
+
+    const requiredParagraph =
+      this.formRequiredFieldsParagraph.createParagraphRequiredFields();
+    container.appendChild(requiredParagraph);
+
+    return container;
+  };
+
+  createSubmitButton = (container: any): any => {
+    const buttonCreator = new ButtonCreation();
+    const successMessage = this.formSuccessMessage.showSuccessMessage();
+      const clickHandler = async () => {
+      if (this.validateForm()) {
+        try {
+          const formData = new FormData(
+            document.querySelector("#contact-form") as HTMLFormElement
+          );
+          console.log(formData);
+          const formObject: { [key: string]: any } = {};
+          formData.forEach((value, key) => {
+            formObject[key] = value;
+          });
+
+          const response = await fetch("https://api.web3forms.com/submit", {
+            method: "POST",
+            headers: {
+              "Content-Type": "application/json",
+              Accept: "application/json",
+            },
+            body: JSON.stringify({
+              ...formObject,
+              access_key: "YOUR_GENERATED_API_KEY",
+            }),
+          });
+
+          const result = await response.json();
+
+          if (result.success && container) {
+            container.appendChild(successMessage);
+            console.log("Form submitted successfully!");
+            (
+              document.querySelector("#contact-form") as HTMLFormElement
+            ).reset();
+          } else {
+            alert("Form submission failed: " + result.message);
+          }
+        } catch (error) {
+          if (error instanceof Error) {
+            alert("An error occurred: " + error.message);
+          } else {
+            alert("An unknown error occurred.");
+          }
+        }
+      }
+    };
+    const containerButton = new DivCreatorWithClassName().createDiv(classNames.buttonContainer);
+    const sendButton = config.buttons.find((button) => button.name === "send");
+    const sendButtonClass = sendButton?.class;
```

Comment:
to powinno byc wyciagane w srodku ifa bo tutaj to bez sensu bo jak nie istnieje button to po co wyciagac